jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -10,13 +10,81 @@
+from jdaviz.core.user_api import PluginUserApi
 from jdaviz.core.tools import ICON_DIR
 
 __all__ = ['PlotOptions']
 
 
 @tray_registry('g-plot-options', label="Plot Options")
 class PlotOptions(PluginTemplateMixin):
+    """
+    The Plot Options Plugin gives access to per-viewer and per-layer options and enables
+    setting across multiple viewers/layers simultaneously.
+
+    Only the following attributes and methods are available through the
+    :ref:`public plugin API <plugin-apis>`:
+
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.show`
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.open_in_tray`
+    * ``multiselect``:
+      whether ``viewer`` and ``layer`` should both be in multiselect mode.
+    * ``viewer`` (:class:`~jdaviz.core.template_mixin.ViewerSelect`):
+    * ``layer`` (:class:`~jdaviz.core.template_mixin.LayerSelect`):
+    * :meth:`select_all`
+    * ``layer_visible`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`)
+    * ``subset_visible`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      whether a subset should be visible.
+    * ``subset_color`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not exposed for Specviz
+    * ``axes_visible`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`)
+    * ``collapse_function`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      only exposed for Cubeviz
+    * ``line_color`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not exposed for Imviz
+    * ``line_width`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not exposed for Imviz
+    * ``line_opacity`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not exposed for Imviz
+    * ``as_steps`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not exposed for Imviz
+    * ``uncertainty`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not exposed for Imviz
+    * ``stretch_func`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not exposed for Specviz
+    * ``stretch_preset`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not exposed for Specviz
+    * ``stretch_vmin`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not exposed for Specviz
+    * ``stretch_vmax`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not exposed for Specviz
+    * ``image_visible`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not exposed for Specviz
+    * ``image_color_mode`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not exposed for Specviz
+    * ``image_color`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not exposed for Specviz.  This only applies when ``image_color_mode`` is "Monochromatic".
+    * ``image_cmap`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):

```

Comment:
Might be slightly clearer to spell it out unless it is already taken by something else in glue?
```suggestion
    * ``image_colormap`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
```

Reply:
ok, I think I'm in favor of spelling it out too since we do in the label in the UI (even though cmap is very common).